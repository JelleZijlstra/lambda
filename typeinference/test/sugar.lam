let f a b = a + b

;;

f 1 2
