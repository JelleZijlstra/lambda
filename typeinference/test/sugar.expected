%%stdout%%
Result: module
	let f = \a. \b : 'typevar1. a + b
	let __result__ = 3
end
%%stderr%%
